src/RuNNer.jl

```diff
@@ -128,8 +170,15 @@ end
 for writing a series of trajectories in a single runner input.data file. Output is an IOStream for use in writing configs.
```

Comment:
what is the usage of third method for writefile, what is 'ix'
I am a bit at a loss here: testconfig seems to be a position only (not a Config) but then it become an SVector?